desktop/libs/aws/src/aws/conf_tests.py

```diff
@@ -66,9 +67,39 @@ def test_has_s3_access_when_raz_enabled(self):
     # When RAZ is not enabled
     assert_false(conf.has_s3_access(self.user))
 
-    reset = RAZ.IS_ENABLED.set_for_testing(True)
+    # When RAZ is enabled for S3
+    resets = [
+      RAZ.IS_ENABLED.set_for_testing(True),
+      conf.AWS_ACCOUNTS.set_for_testing({'default': {
+        'region': 'us-west-2',
+        'host': 's3-us-west-2.amazonaws.com',
+        'allow_environment_credentials': 'false'
+      }})
+    ]
     try:
       assert_true(conf.has_s3_access(self.user))
     finally:
-      reset()
+      for reset in resets:
+        reset()
+      conf.clear_cache()
+
+  def test_is_raz_s3(self):
+
+    # When RAZ is not enabled
+    assert_false(conf.is_raz_s3())
+
+    # When RAZ is enabled for S3
+    resets = [
+      RAZ.IS_ENABLED.set_for_testing(True),
+      conf.AWS_ACCOUNTS.set_for_testing({'default': {
+        'region': 'us-west-2',
+        'host': 's3-us-west-2.amazonaws.com',
+        'allow_environment_credentials': 'false'
+      }})
```

Comment:
PS: better test would have been to test first with Raz enabled only that we still are on False